fdbserver/include/fdbserver/DataDistributionTeam.h

```diff
@@ -69,9 +69,41 @@ FDB_DECLARE_BOOLEAN_PARAM(ForReadBalance);
 FDB_DECLARE_BOOLEAN_PARAM(PreferLowerReadUtil);
 FDB_DECLARE_BOOLEAN_PARAM(FindTeamByServers);
 
+class TeamSelect {
+public:
+	enum Value : int8_t {
+		INVALID = 0,
+		AVOIDMOVEMENT, // Avoid data movement by selecting srcTeam if it's not busy
+		WANTSRCSERVERS, // Ask for srcTeam
+		WANTTRUEBEST, // Ask for the most or least utilized team
+		OTHER
+	};
+	TeamSelect() : value(INVALID) {}
+	TeamSelect(Value v) : value(v) { ASSERT(v != INVALID); }
+	std::string toString() const {
+		switch (value) {
+		case AVOIDMOVEMENT:
+			return "Avoid_Movement";
+		case WANTSRCSERVERS:
+			return "Want_SrcServers";
+		case WANTTRUEBEST:
+			return "Want_TrueBest";
+		case OTHER:
+			return "Other";
+		default:
```

Comment:
If we keep the default `TeamSelect()` constructor with `INVALID` (see comment above): then I think we also need a case for `INVALID` since that's what the `value` is initialized to, so we shouldn't ASSERT fail for an expected case. Or, if we expect that `toString()` should never be called on a default `TeamSelect`, then we need to make that super clear with comments.

Reply:
moved out `INVALID`